projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -9435,6 +9435,11 @@ FORWARD_SLASH
    '/'
 ;
 
```

Comment:
It did not break any of the existing unit tests or parsing of test configs.

There is indeed a similar DIGIT: F_Digit; rule (line 9402). Checked with ANTLR4 documentation that when a terminal (like one of 0 through 9) matches two lexical rules, it is fine, with only restriction that the terminals themselves used directly (e.g. as in xyz: 1;) will be disallowed, but using corresponding LHS components (like F_Digit and F_HexDigit would be fine).

Can you please elaborate on what is going to be broken, and additional tests could be added to cover that, and in which case, F_Digit: DIGIT; case would also need a fix?